src/Contracts/DocumentsQuery.php

```diff
@@ -56,13 +56,34 @@ public function hasFilter(): bool
         return isset($this->filter);
     }
 
+    /**
+     * Prepares fields for request
+     * Fix for 1.2 document/fetch.
+     *
+     * @see https://github.com/meilisearch/meilisearch-php/issues/522
+     *
+     * @return array|string|null
+     */
+    private function fields(): mixed
```

Comment:
The `mixed` broke the implementation it is only introduced in PHP 8 right? So we can't use that, because we still support php 7.4

```suggestion
    private function fields()
```

Reply:
@brunoocasali yep, my bad. I'll fix it